### PR TITLE
Warn users when they meteor reset with MONGO_URL.

### PR DESCRIPTION
**Pull request by laosb**
#1469 , an issue long long ago.

### CODE REVIEW

File: tools/cli/commands.js [hunk 1]
```diff
@@ -1190,6 +1190,13 @@ main.registerCommand({
       Console.command("meteor deploy appname"), Console.options({ indent: 2 }));
     return 1;
   }
+  
+  if (process.env.MONGO_URL) {
+    Console.error("meteor reset won't reset the DB provided by MONGO_URL, as it can be dangerous " +
+                  "if you specify MONGO_URL to your production server. If you are sure to do that, " +
+                  "do it yourself by raw mongo queries.");
+    return 1;
```
Comment (abernix) [inline tools/cli/commands.js]: `meteor reset` does other project reset stuff besides Mongo.  Should it  be allowed to do the rest of it's work and just give a `Console.info` warning that it *didn't* clean the database?

Comment (laosb) [inline tools/cli/commands.js]: Oh yeah, maybe.

**Comment by laosb** [comment]
@abernix Done!

File: tools/cli/commands.js [hunk 2]
```diff
@@ -1192,10 +1192,8 @@ main.registerCommand({
   }
   
   if (process.env.MONGO_URL) {
-    Console.error("meteor reset won't reset the DB provided by MONGO_URL, as it can be dangerous " +
-                  "if you specify MONGO_URL to your production server. If you are sure to do that, " +
-                  "do it yourself by raw mongo queries.");
-    return 1;
+    Console.info("meteor reset won't reset the DB provided by MONGO_URL, while other project" +
+                 "things get reset. If you are sure to reset DB, do it yourself by raw mongo queries.");
```
Comment (abernix) [inline tools/cli/commands.js]: And I would personally recommend this for the text, but maybe wait for more input before you change the PR:
```
As a precaution, `reset` only clears the local, Meteor-provided database.  The database specified with `MONGO_URL` will *NOT* be reset and would need to be reset manually.
```

Comment (zol) [inline tools/cli/commands.js]: I like @abernix's text better and would like it to be further refined to

`As a precaution, `meteor reset` only clears the local database that is provided by `meteor run` for development. The database specified with `MONGO_URL` will *NOT* be reset.`

**Comment by zol** [comment]
Happy to merge this once the text is updated.

**Comment by laosb** [comment]
Updated :wink:

**Comment by zol** [comment]
Thanks!

**Pull request merged**